templates/typescript-ciya-auth-mcp/packages/web-app/src/features/chat/clickhouse-tool-invocation.tsx

```
@@ -0,0 +1,386 @@
+                            key={column}
+                            className="p-1.5 text-left align-middle font-semibold text-muted-foreground whitespace-nowrap"
+                          >
+                            {column}
+                          </th>
+                        ))}
+                      </tr>
+                    </thead>
+                    <tbody className="[&_tr:last-child]:border-0">
+                      {queryData.map((row: any, rowIndex: number) => (
+                        <tr
+                          key={rowIndex}
```

Biome (2.4.4)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)